lib/postmark/message_extensions/mail.rb

```diff
@@ -15,6 +15,22 @@ def tag=(val)
       header['TAG'] = val
     end
 
+    #
+    # @val accepts :html_only, :text_only, :html_and_text, :none
+    #
+    def track_links(val = nil)
+
```

Comment:
We don’t leave blank lines after method definition anywhere in the gem. Please try to comply with existing formatting.